posthog/api/test/test_dashboard.py

```diff
@@ -165,6 +165,29 @@ def test_no_cache_available(self):
         self.assertEqual(response["items"][0]["result"], None)
         self.assertEqual(response["items"][0]["last_refresh"], None)
 
+    def test_refresh_cache(self):
+        dashboard = Dashboard.objects.create(team=self.team, name="dashboard")
+        filter_dict = {
+            "events": [{"id": "$pageview"}],
+            "properties": [{"key": "$browser", "value": "Mac OS X"}],
+        }
+
+        with freeze_time("2020-01-04T13:00:01Z"):
+            # Pretend we cached something a while ago, but we won't have anything in the redis cache
+            item = DashboardItem.objects.create(
+                dashboard=dashboard, filters=Filter(data=filter_dict).to_dict(), team=self.team, last_refresh=now()
+            )
+
+        with freeze_time("2020-01-20T13:00:01Z"):
+            response = self.client.get("/api/dashboard/%s?refresh=true" % dashboard.pk).json()
+
+            self.assertIsNotNone(response["items"][0]["result"])
+            self.assertIsNotNone(response["items"][0]["last_refresh"])
+            self.assertEqual(response["items"][0]["result"][0]["count"], 0)
```

Comment:
Maybe worth adding an assertion to before the refresh happened to check this number was actually different and therefore the cache indeed changed?